### PR TITLE
Fix local binary rebuilding.

### PR DESCRIPTION
**Pull request by wbxyz**
The TUI was building binaries that were dynamically linked, and that broken when they were copied into the rebuilder container.

This was really annoying to debug because the error message didn't mention anything about how the binary was built, it just said `exec ./rebuilder: no such file or directory` even though the binary had been sucessfully copied.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Locally built binaries
```
Comment (msuozzo) [inline .gitignore:1]: i don't think we can realistically keep this in sync. i think it's preferable to set the output location for the build to something like /tmp so it doesn't create the binary in the repo at all.

Comment (wbxyz) [reply .gitignore:1]: I think the only thing we currently build is this binary, so it's fine for now? But yeah I agree that it's not great to pollute the local dir and we should eventually clean that up.

Comment (msuozzo) [reply .gitignore:1]: i'm not crazy about adding application-specific names to the gitignore. what would the process be for changing the write location for the binary?

Comment (wbxyz) [reply .gitignore:1]: I tweaked the build libraries to use a `./bin/` dir instead.

File: build/binary/binary.go [hunk 2]
```diff
@@ -18,20 +18,22 @@ package binary
...
 	"log"
+	"os"
 	"os/exec"
 	"path/filepath"
 )
 
 // Build constructs a binary for one of the project's microservices.
 func Build(ctx context.Context, name string) (path string, err error) {
-	cmd := exec.CommandContext(ctx, "go", "build", "-o", name, "./cmd/"+name)
+	cmd := exec.CommandContext(ctx, "go", "build", "-o", "./bin/"+name, "./cmd/"+name)
```
Comment (msuozzo) [inline build/binary/binary.go:28]: filepath.Join?

**Review by msuozzo — APPROVED** [review]

File: build/binary/binary.go (hunk 2 above)
Comment (wbxyz) [reply build/binary/binary.go]: Done.

**Review by msuozzo — APPROVED** [review]

**Pull request merged**